Source/CMakeLists.txt

```diff
@@ -201,6 +204,14 @@ else() # Otherwise include only chosen ones
 	endif()
 endif()
 
+if (INCLUDE_IMGUI_PLUGIN)
+	# if true, then git clone the imgui plugin (with recursive submodules). it
```

Comment:
Can probably get rid of this comment, as we have it in the file itself

Reply:
done